src/scripts/createCoreInstance.ts

```diff
@@ -58,6 +67,8 @@ export const createCoreInstance = async (environment: Environment) => {
 
   const blockchainDate = new BlockchainDate(environment.BLOCKCHAIN_HTTP_URL)
 
+  const logger = environment.LOG_FILE ? loggerFile : loggerConsole
```

Comment:
```suggestion
  const logger = environment.LOG_FILE ? tracer.dailyfile({
  root: './storage/logs',
  maxLogFiles: 10,
  allLogsFileName: 'log',
  level: 0
}) : tracer.colorConsole()
```

Instanced only if chosen